sql/step_04_all_following_steps.sql

```diff
@@ -84,3 +84,4 @@ source step_87_alter_table_opd.sql;
 source step_88_alter_table_patient.sql;
 source step_89_alter_tables_innodb.sql;
 source step_90_login_retries.sql;
+source step_91_oh_update_table_laboratory.sql;
```

Comment:
```suggestion
source step_91_alter_laboratory_with_status.sql;
```
There is no need to add "_oh_" to the file name and it is better to include the changes made in the file name.  `update_table_laboratory` is too generic.

Reply:
suggested: `step_91_add_laboratory_status.sql`